### PR TITLE
Bugfix: 'send transaction' form validation

### PR DESCRIPTION
**Pull request by kalgoop**
Previous implementation was wrong.
Also, accessing 'valid' attribute to check for errors is wrong, as valid attribute has 'undefined' value.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by alexbarnsley** [comment]
Hi, thanks for your Pull Request. It would help those reviewing if you could describe how the previous implementation was wrong, otherwise there isn't really any way to compare. Thanks

File: src/pages/transaction/transaction-send/transaction-send.ts [hunk 1]
```diff
@@ -199,16 +199,13 @@ export class TransactionSendPage implements OnInit {
...
-      const control = this.sendTransactionHTMLForm.form.controls[name];
-      isValid = control.valid;
-      if (!isValid) {
-        break;
-      }
+    let isValid: boolean = true;
+    
+    if (
+      this.sendTransactionHTMLForm.form.controls['amount'].value <= 0
+      || this.sendTransactionHTMLForm.form.controls['smartBridge'].length > 60
```
Comment (alexbarnsley) [inline src/pages/transaction/transaction-send/transaction-send.ts]: The smart bridge length can be upto 64 characters, not 60

**Comment by kalgoop** [comment]
valid attribute was returning null each time.
In the number type input, constraints like min max dont work in ionic. 
So it was allowing negative values of ARK to be sent (transaction request was being sent from app)

**Comment by dated** [comment]
The smartbridge is delimited to 60 also here: https://github.com/ArkEcosystem/mobile-wallet/blob/master/src/pages/transaction/transaction-send/transaction-send.html#L70.

The validation could be also fixed by simple adding validators to the FormGroup: https://github.com/ArkEcosystem/mobile-wallet/blob/master/src/pages/transaction/transaction-send/transaction-send.ts#L303, i.e. `Validators.min()` and `Validators.max()`.

**Comment by kalgoop** [comment]
@dated I was trying to do the same. But I couldn't figure out how to add commits to an existent PR, if you deleted the fork (I deleted the fork from my repos)

**Comment by dated** [comment]
> @dated I was trying to do the same. But I couldn't figure out how to add commits to an existent PR, if you deleted the fork (I deleted the fork from my repos)

Hmm. Tough one. I don't think you can edit the source of a PR. If you still have the branch locally you can try to refork and repush it, maybe GitHub picks up the reference.

**Comment by kalgoop** [comment]
#227 I have put a new PR for this only, since I had deleted the branch associated with this PR and no modifications were possible in this PR.

**Pull request closed without merging**